CHANGELOG.md

```diff
@@ -1840,6 +1840,34 @@ Other minor changes
   ∈-++↔++ : ∀ {v} xs ys → v ∈ xs ++ ys ↔ v ∈ ys ++ xs
   ```
 
+* Exposed container combinator conversion functions from `Data.Container.Combinator.Properties` separate from their correctness proofs in `Data.Container.Combinator`:
+  ```
+  to-id      : ∀ {s p x} {X : Set x} → F.id X → ⟦ id {s} {p} ⟧ X
```

Comment:
Also we tend to omit implicit arguments in the types in the CHANGELOG so that we give only the most important bits of the proof. For example this would become `F.id X → ⟦ id ⟧ X`